physionet-django/project/modelcomponents/storage.py

```diff
@@ -68,6 +68,21 @@ def public_s3_uri(self):
         """
         return f's3://{self.bucket_name}/{self.project.slug}/{self.project.version}/'
 
+    def user_included_in_ap_policy(self, user):
```

Comment:
i think it is clearer to include "access_point" here. e.g. `user_in_access_point_policy`